chart/templates/tap-rbac.yaml

```diff
@@ -0,0 +1,40 @@
+{{with .Values -}}
+---
+###
+### Tap RBAC
+###
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: linkerd-{{.Namespace}}-tap
+rules:
+- apiGroups: [""]
+  resources: ["pods", "endpoints", "services", "replicationcontrollers", "namespaces"]
```

Comment:
I'm pretty sure the tap server doesn't deal with the `endpoints` API, so you can remove it from this list.

Reply:
Thanks! wasn't quite sure how I could determine which resources were used by each control plane component, but I think the code below is a good indicator. https://github.com/linkerd/linkerd2/blob/f758a9e4282b2e181fcd06ecb20a44f215ae7e91/controller/cmd/tap/main.go#L27

Reply:
Yep, that's how I'd determine it